artifact/v3/header/artifact_scripts.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+
+#include <artifact/v3/header/header.hpp>
+
+#include <vector>
+
+namespace mender {
+namespace artifact {
+namespace v3 {
+namespace header {
+namespace artifact_scripts {
+
+ExpectedArtifactScript Parse(io::Reader &reader) {
+	log::Info("TODO - Parse the scripts names - and verify");
+	// Read the whole script to a string, and store it in memory
+	vector<uint8_t> buf(4096); // TODO - Use a stream, and variable size
```

Comment:
I take it this means that the file storing is not part of this PR?

Reply:
Nope, removed.